config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_dili.yml

```diff
@@ -578,155 +630,169 @@ patient:
     categories:
     - biolink:Disease
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id142
+    enum:
     - '0'
     - '1'
     type: string
     search_term: gastrointestinal disease
+    limit: 3
   GilbertsSyndromeDx:
     categories:
     - biolink:PhenotypicFeature
     - biolink:Disease
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id143
+    enum:
     - '0'
     - '1'
     type: string
     search_term: gilberts syndrome
+    limit: 1
   HBV_Positive:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id145
+    enum:
     - '0'
     - '1'
     type: integer
     search_term: hepatitis B virus infection
+    limit: 1
   HBeAg:
     categories:
     - biolink:PhenotypicFeature
     - biolink:DiseaseOrPhenotypicFeature
-    enum: &id146
+    enum:
     - '0'
     - '1'
     type: string
-    search_term: HB3 antigen, hepatitis B antigen
+    search_term: 
+    limit: 

```

Comment:
I would _prefer_ it if we didn't have any blank search terms and limits, but let me know if you want to leave these in for a reason and I'll just need to add checks for this in the pre-compute script.

Reply:
I included blank search_term and limit parameters for variables that Name Resolver _should_ be able to respond to but currently does not. I left the parameters blank as a reminder to myself to (1) post tickets to Name Resolver's GitHub repo and (2) fill in the blanks when available. Do you mind if we leave them that way?

Reply:
Yep! That's a good reason.